spec/spec.md

```diff
@@ -0,0 +1,174 @@
+# Verona Language Spec
+
+## Lexical Elements
+
+### Comments
+
+### Identifiers and Symbols
+
+- `_` as "don't care".
+
+### Strings
+
+### Constants
+
+- Boolean
+- Various integers and floats
+
+> How to talk about integer literals not having a type yet?
```

Comment:
Someone somewhere surely has said something useful about Haskell's handling of (initially numeric, and now also string and list) literals.  https://www.haskell.org/tutorial/numbers.html is surely someone and somewhere, but as to something useful, I'll leave judgement to the reader.